elements/lisk-p2p/src/errors.ts

```diff
@@ -98,9 +98,11 @@ export class RequestFailError extends Error {
 		this.errorResponse = errorResponse || new Error(message);
 		this.peerId = peerId || '';
 		this.peerVersion = peerVersion || '';
-		this.message = `${this.message}: PeerId: ${this.peerId}: Peer Version: ${
-			this.peerVersion
-		}`;
+		this.message = peerId
```

Comment:
Shouldn't peerId always exist?

Reply:
As we discussed that it's being used in other places of the p2p-lib where we don't use peerId so we will keep it optional

Reply:
`this.message = peerId` seems very weird. shouldn't it be message?

Reply:
That's a conditional statement, when `peerId` is available then we build the message